addons/website_livechat/static/tests/tours/website_livechat_chatbot_flow.js

```diff
@@ -1,200 +1,260 @@
+            },
+            {
+                // should ask for website now
+                trigger: messagesContain("Would you mind providing your website address?"),
+            },
+            {
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run: "edit https://www.fakeaddress.com",
+            },
+            {
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run: "press Enter",
+            },
+            {
+                trigger: messagesContain(
+                    "Great, do you want to leave any feedback for us to improve?"
+                ),
+                // should ask for feedback now
+            },
+            {
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run: "edit Yes, actually, I'm glad you asked!",
+            },
+            {
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run: "press Enter",
+            },
+            {
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run: "edit I think it's outrageous that you ask for all my personal information!",
+            },
+            {
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run: "press Enter",
+            },
+            {
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run: "edit I will be sure to take this to your manager!",
+            },
+            {
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run: "press Enter",
+            },
+            {
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run: "edit I want to say...",
+            },
+            {
+                // Simulate that the user is typing, so the chatbot shouldn't go to the next step
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run() {
+                    let counter = window.debounceAnswerCount;
+                    const target = new TourHelpers(this.anchor);
+                    const delay =
+                        odoo.__WOWL_DEBUG__.root.env.services["mail.store"].Chatbot
+                            .MULTILINE_STEP_DEBOUNCE_DELAY_TOUR;
+                    setTimeout(() => {
+                        target.edit("Never mind!");
+                        setTimeout(() => {
+                            if (window.debounceAnswerCount <= counter) {
+                                console.error(
+                                    "Chatbot should stay in multi line step when user is typing."
+                                );
+                            }
+                            counter = window.debounceAnswerCount;
+                        }, delay);
+                    }, 100);
```

Comment:
Why the `setTimeout(100)`? This looks useless, just make the test more prone to race conditions